exe/dash_timeline_validator

```diff
@@ -1,10 +1,34 @@
 #!/usr/bin/env ruby
 
+require "optparse"
 require "dash_timeline_validator"
 
+options = {}
+OptionParser.new do |opts|
+    opts.banner = "DASH Timeline Validator"  
```

Comment:
Banner should follow the model:

```
DASH Timeline Validator 0.1.2 (2019 Apr 12)

usage: dash_timeline_validator uri [arguments] 
   or: dash_timeline_validator [arguments] --verify_segments_duration=true          check the duration of every segment when setted to true
[...]

Arguments:
   --verify_segments_duration			Check the duration of every segment when setted to true
   [...]
```


Also, the banner should be shown when no arguments is passed.

Reply:
Done